### PR TITLE
Feature/ Create card component

### PR DESCRIPTION
**Pull request by JennyDoJD**
- Design
image

- Result
image
image
image

- Ticket: [Create components](https://trello.com/c/bvrOqz8S/42-create-components)

### CODE REVIEW

**Comment by huyencao-agilityio** [comment]
@JennyDoJD Conflict, please help to check

File: practice/src/index.html [hunk 1]
```diff
@@ -6,8 +6,47 @@
...
+      <div>
+        <figure class="card-image">
+          <img src="#" alt="food image" />
+        </figure>
+        <div class="card-title">
+          <p class="main-title card-name">Spicy seasoned seafood noodles</p>
+          <div class="card-desc">
+            <p class="main-desc">$ 2.29</p>
+            <div class="circle"></div>
+            <p class="main-desc">20 Bowls</p>
```
Comment (huyencao-agilityio) [inline practice/src/index.html:26]: Should be quantity

File: practice/src/index.html [hunk 2]
```diff
@@ -6,8 +6,47 @@
...
+          <use xlink:href="./assets/images/icons/icons.svg#delete-icon" />
+        </svg>
+      </div>
+      <div>
+        <figure class="card-image">
+          <img src="#" alt="food image" />
+        </figure>
+        <div class="card-title">
+          <p class="main-title card-name">Spicy seasoned seafood noodles</p>
+          <div class="card-desc main-desc">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should define the common class first, `<div class="main-desc card-desc">`

File: practice/src/index.html [hunk 3]
```diff
@@ -6,8 +6,47 @@
...
+            <p>$ 2.29</p>
+            <div class="circle"></div>
+            <p>20 Bowls</p>
+          </div>
+        </div>
+      </div>
+      <div>
+        <button class="btn btn-edit">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should be a link, not button

File: practice/src/css/components/card.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.card-desc {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+}
+
+.btn-edit {
```
Comment (huyencao-agilityio) [inline practice/src/css/components/card.css]: Should move this style to button component

File: practice/src/index.html (hunk 2 above)
Comment (huyencao-agilityio) [reply practice/src/index.html]: Please douple check

Comment (JennyDoJD) [reply practice/src/index.html]: I have fixed it

File: practice/src/index.html [hunk 5]
```diff
@@ -6,30 +6,67 @@
...
+      </div>
+      <div>
+        <figure class="card-image">
+          <img src="#" alt="food image" />
+        </figure>
+        <div class="card-title">
+          <p class="tertiary-title card-name">Spicy seasoned seafood noodles</p>
+          <div class="main-desc card-desc">
+            <p>$ 2.29</p>
+            <div class="circle"></div>
```
Comment (canhpham-zb) [inline practice/src/index.html]: @JennyDoJD Why should we need this div? can you use special character?

File: practice/src/index.html [hunk 6]
```diff
@@ -6,30 +6,67 @@
...
+          <use xlink:href="./assets/images/icons/icons.svg#delete-icon" />
+        </svg>
+      </div>
+      <div>
+        <figure class="card-image">
+          <img src="#" alt="food image" />
+        </figure>
+        <div class="card-title">
+          <p class="tertiary-title card-name">Spicy seasoned seafood noodles</p>
+          <div class="main-desc card-desc">
```
Comment (canhpham-zb) [inline practice/src/index.html:23]: @JennyDoJD why do we need `main-desc`? Pls consider before you create new class name

**Review by canhpham-zb — CHANGES_REQUESTED** [review]

File: practice/src/index.html (hunk 6 above)
Comment (JennyDoJD) [reply practice/src/index.html:23]: it is a common class

Comment (ghost) [reply practice/src/index.html:23]: @JennyDoJD could you share css file contain this class?

Comment (JennyDoJD) [reply practice/src/index.html:23]: [https://github.com/JennyDoJD/javascript-training/blob/feature/create-text-component/practice/src/css/components/text.css](https://github.com/JennyDoJD/javascript-training/blob/feature/create-text-component/practice/src/css/components/text.css)

**Review by ghost — APPROVED** [review]

**Pull request merged**